app/ConfigPages.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+// Set the number of pages for post and comment views 
+define("POST_PER_PAGE", 4);
```

Comment:
Tu as une syntaxe plu simple :
 ```suggestion
const POST_PER_PAGE =  4;
```

Reply:
constantes ajoutées